lottie-ios/Classes/PublicHeaders/LOTAnimationView.h

```diff
@@ -46,9 +46,12 @@ typedef void (^LOTAnimationCompletionBlock)(BOOL animationFinished);
 
 // TODO
 /// Sets a progress from 0 - 1 of the animation. If the animation is playing it will stop and the compeltion block will be called.
+/// The animation progress is in terms of absolute progress of the defined animation and does not
+/// take into account negative speeds.
```

Comment:
Sure.

Just a thought... if I call play, the progress goes from 0 to 1 (regardless of play direction) where as setting the value directly will cause a different behaviour so the property does different things depending on if the animation is playing or not.